### PR TITLE
[FIX] elastic_service: Add support for 'in' operator into scoped queries and allows raw_es_query

### PR DESCRIPTION
**Pull request by lmignon**
```
{%with_scope raw_es_query: {filter:{terms:{attributes.color:['red', 'yellow']}}} %}
```

### CODE REVIEW

File: lib/shop_invader/services/elastic_service.rb [hunk 1]
```diff
@@ -142,6 +142,8 @@ def build_params(conditions)
               end
             elsif %w(gt gte lt lte).include?(op)
               params[:bool][:filter] << { range: { name => { op => value } } }
+            elsif %w(in).include?(op)
```
Comment (rvalyi) [inline lib/shop_invader/services/elastic_service.rb]: not a big deal but %w is to make list of strings. For a single string IMHO it is cleaner to have
`elsif op == "in"`

Comment (lmignon) [reply lib/shop_invader/services/elastic_service.rb]: @rvalyi I'm not a ruby developer :smirk: I'll change the code

Comment (lmignon) [reply lib/shop_invader/services/elastic_service.rb]: @rvalyi done

**Review by sebastienbeau — APPROVED** [review]
Ok for me thanks the fix and the improvement, it's great to be able to do directly "raw" request

**Pull request merged**